plugins/ProfileExtender/class.profileextender.plugin.php

```diff
@@ -576,15 +576,19 @@ public function utilityController_exportProfiles_create($sender) {
 
         $i = 0;
         foreach ($fields as $slug => $fieldData) {
+            // Don't overwrite Location data (it's always stored in the user table, never the userMeta table).
+            if (strtolower($slug) === 'location') {
```

Comment:
I think you would be better to check against the `$columnNames` variable where `"Location"` is added above and then you will prevent bugs for any column that has already been added.

Reply:
Make sure to use a similar case-insensitive comparison.

Reply:
It's worth noting that this still may not work for all column names because of some other behaviors --  for instance, creating a profile extender field called "Points" automatically ends up creating a "Points1" field, so you end up getting this: `$Configuration['ProfileExtender']['Fields']['Points1']['Label'] = 'Points';`. So that field will show up twice, as in the screenshot below. There may be other instances of this (although I think it's unlikely these will create problems). In any case, that's why I went with the narrower fix earlier.

![image](https://user-images.githubusercontent.com/17052116/84801766-94555c00-afcd-11ea-82b7-dc2a14186eb8.png)